relay-event-normalization/Cargo.toml

```diff
@@ -29,6 +29,7 @@ relay-log = { workspace = true }
 relay-protocol = { workspace = true }
 relay-statsd = { workspace = true }
 relay-ua = { workspace = true }
+relay-filter = { workspace = true }
```

Comment:
nit: We should move the utility function to `relay-common` instead of pulling in `relay-filter` here.